.github/workflows/deploy.yml

```
@@ -5,6 +5,8 @@ on:
     branches: [ hugo ]
   pull_request:
     branches: [ hugo ]
+  schedule:
+    - cron: 0 0 * * *
```

Although not necessary, I recommend setting the minute value to something else to avoid a "0:00 burst" (since many other projects may also set theirs to 0:00).

```suggestion
    - cron: 25 0 * * *
```